testing/xcb-util-xrm/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Jakub Skrzypnik <j.skrzypnik@openmailbox.org>
+pkgname=xcb-util-xrm
+pkgver=1.2
+pkgrel=0
+pkgdesc="Utility functions for the X resource manager"
+url="https://github.com/Airblader/xcb-util-xrm/"
+arch="all"
+license="MIT"
+makedepends="libxcb-dev xcb-util-dev util-macros bsd-compat-headers automake autoconf libx11-dev"
+source="https://github.com/Airblader/xcb-util-xrm/releases/download/v1.2/xcb-util-xrm-1.2.tar.gz"
+subpackages="$pkgname-dev"
+
+builddir="$srcdir/$pkgname-$pkgver"
+build() {
+	cd "$builddir"
+	autoreconf -fi
```

Comment:
Unfortunately APKBUILD does not run with `set -e` (yet, it’s on our TODO list to change it), so every statement that may fail, but should not fail, and is not the last statement of the function, should end with `|| return 1`.